WireAuthentication/Sources/WireAuthenticationUI/Views/Root/RootViewModel.swift

```diff
@@ -36,4 +37,8 @@ public final class RootViewModel: ObservableObject, Router {
         path.append(destination)
     }
 
+    public func present(modal: ModalDestination) {
```

Comment:
suggestion: if we accept any destination, you could rename this to

```swift
func presentModally(_ destination: some Hashable)
``

Reply:
removed code